packages/chain-adapters/src/cosmossdk/types.ts

```diff
@@ -88,4 +88,9 @@ export type FeeData = {
   gasLimit: string
 }
 
+export type Token = {
```

Comment:
early feedback, slight nitpick, can we give this a slightly more specific name in this repo? `CosmosSDKToken` perhaps?

Reply:
I personally don't like this naming convention when using aliased imports. `cosmossdk.Token` vs the stuttered `cosmossdk.CosmosSDKToken`, just as my 2 cents.